packages/govuk-frontend/src/govuk/components/radios/radios.yaml

```
@@ -2,12 +2,12 @@ params:
   - name: fieldset
     type: object
     required: false
-    description: Options for the fieldset component (for example legend).
+    description: Can be used to add a fieldset to the radios component.
```

Radios always need a fieldset (unlike checkboxes where there can be only one, like 'I accept the terms and conditions') so maybe reword this to make it sound less optional?

---

Ah this is my fault because the template has `{% if params.fieldset %}`

Do we need to double check?

---

It was the [**Radios** without fieldset](https://govuk-frontend-review.herokuapp.com/components/radios/without-fieldset/preview) example added in https://github.com/alphagov/govuk-frontend/commit/e060420ac7cb44b245db8fedefbbe5342d64a797

Do we need to flip it to `required: true` at the same time?

https://github.com/alphagov/govuk-frontend/blob/3cd8054a7ca9abadedae6283a4da0d0b0c73b792/packages/govuk-frontend/src/govuk/components/radios/radios.yaml#L242-L252

---

This is a tricky one because there are edge cases where a user might want to manually manage the fieldset (like https://github.com/alphagov/govuk-frontend/issues/1079)

So e.g.

```
{% call govukFieldset %}
    <h2>Group of radios</h2>
    {{ govukRadios({
      // ...
    ) }}
    <h2>Another group of radios</h2>
    {{ govukRadios({
      // ...
    ) }}
{% endcall %}
```

I think it makes sense to optimise the description for the general case? In the vast majority of cases, you should provide a fieldset.

---

Will flip the text but keep `required: false`?

```suggestion
    description: Additional options for the fieldset used by the radios component.
```

---

I've dropped the entire "Additional options for…" as we've done in https://github.com/alphagov/govuk-frontend/pull/4450#discussion_r1401799402

```yaml
    description: The fieldset used by the radios component.
```

---

I've dropped the entire "Additional options for…" as we've done in https://github.com/alphagov/govuk-frontend/pull/4450#discussion_r1401799402

Noted in the commit that this one is special (worded as mandatory but not) and linked to this thread

```yaml
    description: The fieldset used by the radios component.
```